### PR TITLE
SDIT-1195 PrisonerStatusController and /offenders/ids

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonerStatusController.java [hunk 1]
```diff
@@ -46,6 +35,7 @@ public class PrisonerStatusController {
             @ApiResponse(responseCode = "200", description = "OK", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = PrisonerInformation.class))}),
             @ApiResponse(responseCode = "401", description = "Unauthorized.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
             @ApiResponse(responseCode = "404", description = "User not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
+    @VerifyOffenderAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
```
Comment (mikehalmamoj) [inline src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonerStatusController.java:38]: Should we be adding SYSTEM_USER here? I thought it was deprecated...

Comment (karenmillermoj) [reply src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonerStatusController.java:38]: We should only be adding it if we're moving it (as in this case).  The only alternative would be to remove the role if the endpoint has been checked and all associated calling clients don't have that role

Comment (mikehalmamoj) [reply src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonerStatusController.java:38]: Oh sorry it's move from the service. I see, thanks 😄

**Review by mikehalmamoj — APPROVED** [review]

File: src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonerStatusController.java (hunk 1 above)
Comment (steverendell) [reply src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonerStatusController.java:38]: Im just leaving it as it is (or was , on the service layer)

**Pull request merged**